### PR TITLE
Rename ClientService to CommandService

### PR DESCRIPTION
**Pull request by dmdashenkov**

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol PTAL

File: server/src/test/java/org/spine3/server/CommandServiceShould.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public void setUp() {
         boundedContexts.add(customersContext);
 
         // Expose two Bounded Contexts via a Client Service.
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/CommandServiceShould.java]: `...via an instance of {@code CommandService}.`

**Comment by armiol** [comment]
@dmdashenkov please see my comment.

File: server/src/test/java/org/spine3/server/CommandServiceShould.java (hunk 1 above)
Comment (dmdashenkov) [inline server/src/test/java/org/spine3/server/CommandServiceShould.java]: Done

**Comment by dmdashenkov** [comment]
@armiol PTAL

**Comment by armiol** [comment]
@dmdashenkov LGTM.

**Pull request merged**